mkosi/__init__.py

```diff
@@ -5143,29 +5175,15 @@ def parse_args(argv: Optional[List[str]] = None) -> Dict[str, argparse.Namespace
     # Make sure all paths are absolute and valid.
     # Relative paths are not valid yet since we are not in the final working directory yet.
     if args_pre_parsed.directory is not None:
-        args_pre_parsed.directory = os.path.abspath(args_pre_parsed.directory)
-        directory = args_pre_parsed.directory
+        directory = args_pre_parsed.directory = args_pre_parsed.directory.absolute()
     else:
-        directory = os.path.abspath(".")
+        directory = Path(".").absolute()
```

Comment:
You can simply put `Path()` here which defaults to the current directory

Reply:
I actually changed that that Path.cwd(), it DTRT.